src/server/game/Guilds/Guild.h

```diff
@@ -54,6 +54,8 @@ enum GuildMisc
     TAB_UNDEFINED                       = 0xFF,
 };
 
+constexpr uint64 GUILD_BANK_MONEY_LIMIT = UI64LIT(0x7FFFFFFFFFFFF);
```

Comment:
https://en.cppreference.com/w/cpp/types/numeric_limits

Reply:
I don't quite understand what change you want me to do.